doc/api/buffer.markdown

```diff
@@ -670,9 +670,15 @@ console.log(buf.toString('ascii'));
   // Prints: Node.js
 ```
 
-### buf.compare(otherBuffer)
+### buf.compare(otherBuffer[, offset[, length[, thisOffset]]])
```

Comment:
Perhaps the order should be tweaked a bit here and have `length` and `thisOffset` swapped to match more closely with `buffer.copy()` which also deals with two buffers?

I also think `offset` might be better renamed to something like `otherOffset`.